styles/XH.scss

```diff
@@ -194,3 +194,7 @@ body.xh-app {
     text-transform: uppercase;
   }
 }
+
+.xh-tablet * {
+  user-select: none !important;
+}
```

Comment:
I would nest (like the other rules within this file) underneath the `body.xh-app` selector. Not sure if that would make a functional difference but still seems correct and consistent.  (So, move up inside braces and add leading ampersand: `&.xh-tablet * {...}`

Is the `!important` required?

Reply:
Will move to that block now. 

The !important comes from the definition in the client app from which this change originated. I'm not sure it's needed, but someone thought it was. Could test in MY dashboard related client app but would need to cut a release and hit the app with my tablet via blackberry access to be sure (which is fine but will take some time)